### PR TITLE
Move BlogApp files to Seed Realm

### PR DESCRIPTION
**Pull request by burieberry**

### CODE REVIEW

File: packages/experiments-realm/BlogPost/ultimate-guide-remote-work.json [hunk 1]
```diff
@@ -29,29 +29,14 @@
           "self": "../BlogApp/ramped"
         }
       },
-      "categories.0": {
```
Comment (lukemelia) [inline packages/experiments-realm/BlogPost/ultimate-guide-remote-work.json:32]: Was the .0 .1 thing an old format?

**Review by lukemelia — APPROVED** [review]
Looks good once green

File: packages/experiments-realm/BlogPost/ultimate-guide-remote-work.json (hunk 1 above)
Comment (burieberry) [reply packages/experiments-realm/BlogPost/ultimate-guide-remote-work.json:32]: if value is null, it's `categories` only

Comment (lukemelia) [reply packages/experiments-realm/BlogPost/ultimate-guide-remote-work.json:32]: Roger

**Pull request merged**